css/properties.json

```diff
@@ -4712,6 +4640,49 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset": {
+    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": [
+      "offset-distance"
```

Comment:
`offset-position` and `offset-anchor` also allow to set percentage values, so they should be added to the array.

Sebastian